{"uid":"cd8422131fee63e0","name":"Error message for valid credentials","fullName":"Login functionality of ZEN portal: Error message for valid credentials","historyId":"db53cb05f4b9b6667ff3107790fc599c","time":{"start":1751526393273,"stop":1751526397306,"duration":4033},"status":"failed","statusMessage":"AssertionError: Test failed:Expected to stay on login page, but redirected to dashboard\n","statusTrace":"  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/login_steps.py\", line 61, in assertion_failure\n    assert \"login\" in context.driver.current_url, \"Test failed:Expected to stay on login page, but redirected to dashboard\"\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"failed","statusMessage":"AssertionError: Test failed:Expected to stay on login page, but redirected to dashboard\n","statusTrace":"  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/login_steps.py\", line 61, in assertion_failure\n    assert \"login\" in context.driver.current_url, \"Test failed:Expected to stay on login page, but redirected to dashboard\"\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","steps":[{"name":"Given I am on the login page","time":{"start":1751526393273,"stop":1751526396032,"duration":2759},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"When I enter a valid username and password","time":{"start":1751526396033,"stop":1751526397300,"duration":1267},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Then I should see an error message instead of dashboard","time":{"start":1751526397301,"stop":1751526397305,"duration":4},"status":"failed","statusMessage":"AssertionError: Test failed:Expected to stay on login page, but redirected to dashboard\n","statusTrace":"  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/login_steps.py\", line 61, in assertion_failure\n    assert \"login\" in context.driver.current_url, \"Test failed:Expected to stay on login page, but redirected to dashboard\"\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":3,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"severity","value":"normal"},{"name":"feature","value":"Login functionality of ZEN portal"},{"name":"framework","value":"behave"},{"name":"language","value":"cpython3"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[]}],"tags":[]},"source":"cd8422131fee63e0.json","parameterValues":[]}